### PR TITLE
[Instrumentation.Process, Instrumentation.Runtime] Release wtih OTel 1.4.0-rc.4

### PR DESCRIPTION
**Pull request by Kielek**
Fixes N/A

## Changes

Release request
[Instrumentation.Process, Instrumentation.Runtime] Release wtih OTel 1.4.0-rc.4

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* ~~[ ] Design discussion issue #~~
* ~~[ ] Changes in public API reviewed~~

### CODE REVIEW

**Comment by Kielek** [comment]
@xiang17, fyi

File: src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.1.0-beta.5
```
Comment (cijothomas) [inline src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: @xiang17 Do you think we should no be doing an RC, and we do not anticipate any further changes to this package?

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: I would consider rc with stable OTel.

**Review by Yun-Ting — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: why not do stable of this package, when OTel 1.4 is stable as well ?

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: I am not against stable with stable OTel.
I do not like idea where RC version can be compiled only with exact package version, not even the range.

Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: it should be range, given OTel .NET said its the last rc, so we should leave upper bound open.

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: I am not complaining, but the same case was for rc.3.

If possible please merge #994 then this PR.

**Review by cijothomas — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md (hunk 1 above)
Comment (xiang17) [reply src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md]: I think doing RC now is a good idea. We should release the RC before we release the stable anyways.

I agree with Piotr. The exact package version is annoying, especially it causes trouble when downstream wants to test against version upgrades.

**Pull request merged**